### PR TITLE
Specify intent of choice of file formats

### LINKED ISSUE #316: Version control or compare changes?
The criteria [Maintain version control](https://standard.publiccode.net/criteria/version-control-and-history.html) has the requirement:

> You SHOULD prefer file formats that can easily be version controlled.

Technically, modern version control software can handle most files with ease. But what we do want to say with this is more precise, we want to be able to compare changes easily, and not only on a technical level, but on a human readable level. Can we make this more clear in the requirement or is this nitpicky and actually obvious?

### PR DESCRIPTION
**Pull request by Ainali**
Fixes #316

-----
[View rendered criteria/version-control-and-history.md](https://github.com/publiccodenet/standard/blob/ja-meaningful-diffs/criteria/version-control-and-history.md)

### CODE REVIEW

**Review by ericherman — APPROVED** [review]
Is "of" the best word?

Would it be better as "changes *in* the files" or perhaps "changes *within* the files" ....

Regardless of preposition choice, the wording in this diff is an improvement.

**Comment by Ainali** [comment]
I defer to native speakers here and are happy to change. @ElenaFdR any preference between 'in' or 'within'?

File: criteria/version-control-and-history.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ order: 6
 * Every commit message MUST link to discussions and issues wherever possible.
 * You SHOULD group relevant changes in commits.
 * You SHOULD mark different versions of the codebase, for example using revision tags or textual labels.
-* You SHOULD prefer file formats that can easily be version controlled.
+* You SHOULD prefer file formats where the changes of the files can be easily viewed and understood in the version control system.
```
Comment (ElenaFdR) [inline criteria/version-control-and-history.md:15]: 2 options, with slightly different nuance. The first probably fulfills #316 slightly better, but is a clunky sentence.

```
* You SHOULD prefer file formats where the changes within the files can be easily viewed and understood in the version control system.
```
or:

```
* You SHOULD prefer file formats where changes can be easily viewed and understood in the version control system.
```

Comment (ericherman) [reply criteria/version-control-and-history.md:15]: Yeah, I see what you mean. For me, I think the "clunkiness" is probably worth it for the explicitness of fulfilling #316.

**Review by ericherman — APPROVED** [review]

**Pull request merged**